Got a doctest failure... The error is odd.. maybe its transient? I'll push an empty commit to see if it helps.
```
Testing of doctests in the sources finished, look at the results in build/doctest/output.txt.
python: ./Include/internal/pycore_typeobject.h:75: _PyType_GetModuleState: Assertion `et->ht_module' failed.
Aborted (core dumped)
make[1]: *** [Makefile:49: build] Error 134
make[1]: Leaving directory '/home/runner/work/cpython/cpython/Doc'
Testing of doctests in the sources finished, look at the results in build/doctest/output.txt
make: *** [Makefile:127: doctest] Error 1
make: Leaving directory '/home/runner/work/cpython/cpython/Doc'
Error: Process completed with exit code 2.
```

--- a/Doc/library/shutil.rst
+++ b/Doc/library/shutil.rst
@@ -464,11 +464,11 @@ Directory and files operations
    .. versionchanged:: 3.12
       On Windows, the current directory is no longer prepended to the search
       path if *mode* includes ``os.X_OK`` and WinAPI
-      ``NeedCurrentDirectoryForExePathW(cmd)`` is false. When the current
-      directory is a prepended, it is prepended before the *path*.  ``PATHEXT``
-      is used now even when *cmd* includes a directory component or ends with
-      an extension that is in ``PATHEXT``; and filenames that have no extension
-      can be found now.
+      ``NeedCurrentDirectoryForExePathW(cmd)`` is false, else the current
+      directory is prepended even if it is already in the search path;
+      ``PATHEXT`` is used now even when *cmd* includes a directory component
+      or ends with an extension that is in ``PATHEXT``; and filenames that
+      have no extension can now be found.
 
 .. exception:: Error
 

--- a/Doc/library/shutil.rst
+++ b/Doc/library/shutil.rst
@@ -464,9 +464,10 @@ Directory and files operations
    .. versionchanged:: 3.12
       On Windows, the current directory is no longer prepended to the search
       path if *mode* includes ``os.X_OK`` and WinAPI
-      ``NeedCurrentDirectoryForExePathW(cmd)`` is false; ``PATHEXT`` is used
-      now even when *cmd* includes a directory component or ends with an
-      extension that is in ``PATHEXT``; and filenames that have no extension
+      ``NeedCurrentDirectoryForExePathW(cmd)`` is false. When the current
+      directory is a prepended, it is prepended before the *path*.  ``PATHEXT``
+      is used now even when *cmd* includes a directory component or ends with
+      an extension that is in ``PATHEXT``; and filenames that have no extension
       can be found now.
 
 .. exception:: Error

